app/models/series.rb

```diff
@@ -18,6 +18,7 @@ class Series < BaseModel
   has_many :stories, -> { order('episode_number DESC, position DESC, published_at DESC') }
   has_many :schedules
   has_many :audio_version_templates
+  has_many :distributions, as: :distributable
```

Comment:
Why rename?  I thought the Series _was_ "distributable", and the PodcastDistribution is the "distribution of the distributable".